docs/technical_reference/flowsheets/electrodialysis_1stack.rst

```diff
@@ -0,0 +1,81 @@
+
+Introduction
+------------
+Electrodialysis (ED) is a promising technology for desalinating brackish waters and has been deployed at industrial scales [1]_. It utilizes electrical potential to drive ion diffusion through anion and cation exchange membranes, resulting in the dilution of the feed stream while producing a concentrated brine. A single ED stack is an assembly of multiple flow-by cells separated by alternating cation and anion exchange membranes positioned between a pair of electrodes. When a voltage is applied, ions in the cell pair are driven from one channel (forming the diluate channel) to the other (forming the concentrate channel), thereby desalinating water. A one-stack ED desalination system represents a basic ED operation, from which more complicated or larger-scale systems can be derived. Analyzing a one-stack ED system therefore provides valuable information on the technology's performance and cost-effectiveness for a given treatment task.
+
+Equal flow conditions through the diluate and concentrate channels would result in a product water recovery of 50%. This flowsheet simulates a simplest setup of a one-stack ED system without any fluid recirculation, i.e., the ED stack being operated in a in-and-out single direction flow mode. This flowsheet also does not take account of the frictional pressure drop in the channel so no pump is included.  A more complicated ED flowsheet is presented as 
+
+    * `One-stack electrodialysis with concentrate stream recirculation (feed-and-bleed mode) <https://watertap--1385.org.readthedocs.build/en/1385/technical_reference/flowsheets/electrodialysis_1stack_conc_recirc.html>`_.
+
+Implementation
+--------------
+
+The modeled one-stack ED system is illustrated by Figure 1. The feed solution is split into two fluids through a separator unit, entering the diluate and concentrate channels of the ED stack. On the outlet side of the ED stack, all diluate fluids are collected into the total product water, and the concentrate fluids into the total brine stream. Water recovery in this model is volume-based, i.e., the ratio of product volume to total volume of feed solution. The model simulates the steady state of the ED system.
+
+.. figure:: ../../_static/flowsheets/edfs.png
+    :width: 500
+    :align: center
+
+    Figure 1. Flowsheet diagram: one-stack ED system
+
+Documentation for unit models from WaterTAP: 
+    * `Electrodialysis_1D <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/electrodialysis_1D.html>`_
+Documentation for unit models from IDAES: 
+    * `Feed block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
```

Comment:
In other flowsheet documentation, we have not listed these IDAES models. Personally I think they can be left out since these will be present in almost every flowsheet, but I'm curious what others think.